File: client/ios/Contract/Attachment/AttachmentListPayload.swift
import Foundation

public struct AttachmentListPayload: Sendable {
    public let ownerType: String
    public let ownerId: String
    public let count: Int
    public let items: [AttachmentItemPayload]
    public let payloadText: String?

    public init(ownerType: String, ownerId: String, count: Int, items: [AttachmentItemPayload], payloadText: String?) {
        self.ownerType = ownerType; self.ownerId = ownerId; self.count = count; self.items = items; self.payloadText = payloadText
    }
}

File: client/android/Contract/Attachment/AttachmentListPayload.kt
package app.mobiling.client.contract.attachment

/**
 * Marketing America Corp. Oleksandr Tishchenko
 */
data class AttachmentListPayload(
    val ownerType: String,
    val ownerId: String,
    val count: Int,
    val items: List<AttachmentItemPayload>,
    val payloadText: String,
)
